### PR TITLE
test: add notification endp activate status test

### PR DESCRIPTION
**Pull request by karel-rehor**
Adds: 
  * test of activation/deactivation endpoints
  * extra assertions in edit endpoint test

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 51 | :red_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=3649)

**Comment by karel-rehor** [comment]
@subirjolly 
Hi Subir, 
Do you have time to review this?

File: cypress/e2e/shared/notificationEndpoints.test.ts [hunk 1]
```diff
@@ -178,6 +178,10 @@ describe('Notification Endpoints', () => {
         'A minute, an hour, a month. Notification Endpoint is certain. The time is not.'
       const newURL = 'many-faced-god.gov'
 
+      cy.intercept('PUT', `/api/v2/notificationEndpoints/${endpoint.id}`).as(
+        'putEndp'
```
Comment (subirjolly) [inline cypress/e2e/shared/notificationEndpoints.test.ts:182]: Please name this something which makes sense when looking at `cy.wait` and have no context of where this is defined.

**Comment by subirjolly** [comment]
> @subirjolly 
> Hi Subir, 
> Do you have time to review this? 

Left one comment.

**Comment by karel-rehor** [comment]
@subirjolly 

Hi Subir, 
I've made the recommended changes.  Can you please take another look at this PR?

File: cypress/e2e/shared/notificationEndpoints.test.ts [hunk 2]
```diff
@@ -248,6 +272,94 @@ describe('Notification Endpoints', () => {
...
+    if (!t) {
+      throw `Element text: "${elem.text()}" contains no value`
+    }
+    expect(parseInt(t[0]))
+      .to.be.at.least(min)
+      .and.to.be.at.most(max)
+  }
+
+  it('can change endpoint status', () => {
+    cy.intercept('PATCH', '/api/v2/notificationEndpoints/*').as('patchEndp')
```
Comment (subirjolly) [inline cypress/e2e/shared/notificationEndpoints.test.ts:291]: I meant for all the aliases like this..

**Review by subirjolly — CHANGES_REQUESTED** [review]

**Comment by subirjolly** [comment]
> @subirjolly 
> 
> Hi Subir, 
> I've made the recommended changes.  Can you please take another look at this PR? 

Sorry for not being clear earlier. I meant to ask you to change the names of all aliases in this PR to something meaningful.

File: cypress/e2e/shared/notificationEndpoints.test.ts (hunk 2 above)
Comment (karel-rehor) [reply cypress/e2e/shared/notificationEndpoints.test.ts:291]: Sorry, but these types of aliases are part of the code base going back two years and more.  It seems to me this is accepted practice in Influx.

Comment (subirjolly) [reply cypress/e2e/shared/notificationEndpoints.test.ts:291]: i understand that but I am not asking you to change it across the codebase. just here. practices evolve and so should we. rest is upto you, if you don't want to rename this.

**Comment by karel-rehor** [comment]
@subirjolly 

Hi Subir, 

additional changes have been made.  When you have time, can you look at this PR again.

**Review by subirjolly — APPROVED** [review]

**Pull request merged**